src/core/Button/style.ts

```diff
@@ -30,13 +45,15 @@ const ButtonBase = styled(Button, {
 
     const padding = variant === "outlined" ? outlinedPadding : containedPadding;
 
+    const color = colors && colors[colorProp];
```

Comment:
we pick the color object based on `colorProp`, which defaults to `"primary"` from `index.tsx`

Reply:
also I haven't fixed the type error here 😂

Reply:
Fixed! https://github.com/chanzuckerberg/sci-components/pull/111/files#r794982993